src/app/(routes)/createcard/page.tsx

```
@@ -0,0 +1,242 @@
+  const router = useRouter();
+
+  const [formData, setFormData] = useState({
+    photo: null as File | null,
+    photoName: "사진 업로드",
+  });
+
+  const [isLoading, setIsLoading] = useState(false);
+
+  const handleFileChange = (e: ChangeEvent<HTMLInputElement>) => {
+    if (e.target.files && e.target.files[0]) {
+      const file = e.target.files[0];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)